src/components/Footer-Comp/Footer.js

```diff
@@ -1,20 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
 
 import classes from "../Footer-Comp/Footer.module.css";
 
-const Footer = () => (
+function Footer() {
+    const [BlogName] = useState(["CodeMoulding.blog"])
```

Comment:
Names of variables must be `cammelCase` not `CammelCase`, capitalize names are only for Components